assets/xml/objects/object_fu.xml

```diff
@@ -1,52 +1,61 @@
+        <Texture Name="gWindmillManHornAndPipeTex" OutName="horn_and_pipe" Format="ci8" Width="8" Height="8" Offset="0x5EE0" TlutOffset="0x4C60"/>
+        <Texture Name="gWindmillManEyeClosedTex" OutName="eye_closed" Format="ci8" Width="32" Height="32" Offset="0x5F20" TlutOffset="0x4E60"/>
+        <Texture Name="gWindmillManEyeAngryTex" OutName="eye_angry" Format="ci8" Width="32" Height="32" Offset="0x6320" TlutOffset="0x4E60"/>
+        <Texture Name="gWindmillManMouthOpenTex" OutName="mouth_open" Format="ci8" Width="32" Height="16" Offset="0x6720" TlutOffset="0x4E60"/>
+        <Texture Name="gWindmillManMouthAngryTex" OutName="mouth_angry" Format="ci8" Width="32" Height="16" Offset="0x6920" TlutOffset="0x4E60"/>
+        <Texture Name="gWindmillManHairTex" OutName="hair" Format="ci8" Width="8" Height="16" Offset="0x6B20" TlutOffset="0x4E60"/>
+
+        <!-- Guru Guru Limbs -->
+        <Limb Name="gWindmillManTorsoLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_01" Offset="0x6BA0"/>
+        <Limb Name="gWindmillManLeftThighLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_02" Offset="0x6BAC"/>
+        <Limb Name="gWindmillManLeftLegLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_03" Offset="0x6BB8"/>
+        <Limb Name="gWindmillManLeftFootLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_04" Offset="0x6BC4"/>
+        <Limb Name="gWindmillManRightThighLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_05" Offset="0x6BD0"/>
+        <Limb Name="gWindmillManRightLegLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_06" Offset="0x6BDC"/>
+        <Limb Name="gWindmillManRightFootLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_07" Offset="0x6BE8"/>
+        <Limb Name="gWindmillManChestAndMusicBoxPipeLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_08" Offset="0x6BF4"/>
+        <Limb Name="gWindmillManLeftArmAndMusicBoxLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_09" Offset="0x6C00"/>
+        <Limb Name="gWindmillManLimb_006C0C" LimbType="Standard" EnumName="OBJECT_FU_LIMB_0A" Offset="0x6C0C"/> <!--A random triangle near the right side of the music box-->
+        <Limb Name="gWindmillManRightShoulderLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_0B" Offset="0x6C18"/>
+        <Limb Name="gWindmillManRightArmLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_0C" Offset="0x6C24"/>
+        <Limb Name="gWindmillManRightHandAndCrankLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_0D" Offset="0x6C30"/>
+        <Limb Name="gWindmillManHeadLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_0E" Offset="0x6C3C"/>
+        <Limb Name="gWindmillManHornLimb" LimbType="Standard" EnumName="OBJECT_FU_LIMB_0F" Offset="0x6C48"/>
+
+        <!-- Guru Guru Skeleton -->
+        <Skeleton Name="gWindmillManSkel" Type="Flex" LimbType="Standard" LimbNone="OBJECT_FU_LIMB_NONE" LimbMax="OBJECT_FU_LIMB_MAX" EnumName="ObjectFuLimb" Offset="0x6C90" />
```

Comment:
```suggestion
        <Skeleton Name="gWindmillManSkel" Type="Flex" LimbType="Standard" LimbNone="GURU_GURU_LIMB_NONE" LimbMax="GURU_GURU_LIMB_MAX" EnumName="GuruGuruLimb" Offset="0x6C90" />
```
Also the `NONE` and `MAX` limbs, as well as the enum name which has the form `<Prefix>Limb` in PascalCase.